FWCore/python/FWCore/joboptions.py

```diff
@@ -4,11 +4,13 @@ def parse_standard_job_options(scriptname=""):
     """
     Returns the parsed arguments, adding a parser with commonly needed opts:
     - args.nevents      -- number of events (int), specify with --nevents
+    - args.delphescard  -- the input delphes card (string), specify with --delphescard
```

Comment:
The idea was to have "standard" options here and at the same time allow parsers for specific things in the options file. E.g. like [this](https://gist.github.com/jlingema/d170c552bd3caf71a3d713ccf04d5fd5)